src/theme/components/socialmedialinks.ts

```diff
@@ -0,0 +1,185 @@
+// maybe set the border to always be on, but invisible if prop is set to "none"?
+
+import { StyleFunctionProps } from "@chakra-ui/theme-tools";
+
+function getRadius(size) {
+  let radiusSize = "119.125rem"; // 18px @todo Seems to be non standard. Should it be 16px / 1rem / 's'?
```

Comment:
As far as I can tell, this doesn't work for the `borders="circular"` and size medium or large. The "m" and "l" values don't seem to work, but regardless the value "100%" should be used to get the circular border radius for any size.

Reply:
And perhaps a bit more padding? @bigfishdesign13 can better help with this.

Reply:
There are a few things at play here.  First, there is no way to set the border radius as a percentage in Figma and we have to use px values.  That is why you are seeing specific px value in the Figma components.  That being said, Edwin is correct that you should use `"100%"` for the border radius value on the round variants.

Second, the icon in the `"small"` variant is `18px x 18px`, so the border radius in a `px` value needed to be `18px` to get a true round object.  `16px` would have left it a bit short of round.